File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
    let ind= 2
    for(let i=2;i<nums.length;i++)
    {
        if(nums[i]!==nums[ind-2])
        {
            nums[ind++]=nums[i]
        }
    }
    return ind
};

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int inx=2;
        for(int i=2;i<nums.length;i++)
        {
            if(nums[i]!=nums[inx-2])
            {
                nums[inx++]=nums[i];
            }
        }
        return inx;
    }
}